soccer/joystick/GamepadController.cpp

```diff
@@ -41,7 +41,7 @@ GamepadController::GamepadController()
 }
 
 GamepadController::~GamepadController() {
-    QMutexLocker(&mutex());
+    auto lock = lock_mutex();
```

Comment:
you should standardize how you're locking your mutexes...

Reply:
What's non-standardized here? The mutex is actually a private member of the parent now, and I just gave it a protected getter for a lock instead of for the mutex itself.

Reply:
as in like you're using a getter for a lock guard here, but in the other class you're creating the lock guard directly in the function. I mean standardize as in like maybe use either getters or construct the lock guard directly in both cases, so that it's standardized across the codebase